Content.Client/Instruments/InstrumentSystem.cs

```diff
@@ -252,7 +253,7 @@ public bool OpenInput(EntityUid uid, InstrumentComponent? instrument = null)
 
     }
 
-    public bool OpenMidi(EntityUid uid, ReadOnlySpan<byte> data, InstrumentComponent? instrument = null)
+    public bool OpenMidi(EntityUid uid, byte[] data, InstrumentComponent? instrument = null)
```

Comment:
This is a breaking change. IMO it's not worth documenting, just add another overload that takes a span and copies it, but marked `[Obsolete]`.